dingo/experimental_gurobi_functions/update_gurobi_model_experiment.py

```diff
@@ -0,0 +1,41 @@
+import gurobipy as gp
+import numpy as np
+import scipy.sparse as sp
+
+def update_model_constraints_and_bounds(model, Aeq_sparse=None, beq=None, A_sparse=None, b=None,
```

Comment:
Could you plz move this function into the file `dingo/experimental_gurobi_functions/redundant_facet_removal_experiment.py` and delete this file?

Reply:
ok @TolisChal